tests/rigctld.c

```diff
@@ -81,7 +81,7 @@
  *      keep up to date SHORT_OPTIONS, usage()'s output and man page. thanks.
  * TODO: add an option to read from a file
  */
-#define SHORT_OPTIONS "m:r:p:d:P:D:s:S:c:T:t:C:W:w:x:z:lLuovhVZRA:"
+#define SHORT_OPTIONS "m:r:p:d:P:D:s:S:c:T:t:C:W:w:x:z:lLuovhVZRA:b"
```

Comment:
What I see in my working tree is that the master branch lacks `b` but has `n` at the end of the short options list but has `b` at the end of the long options list.

If you haven't added this in another commit, the `bind-all` option should be presented in the "Usage" message (at the bottom of the file) as well.

Whether you choose `b` or `n` for this option shouldn't matter too much.